spec_util/testdata/meld/meld_map_1_map_2_expected.pb.txt

```diff
@@ -0,0 +1,202 @@
+# api_spec.Witness proto
+
+method <
+  id: <
+    api_type: HTTP_REST
+  >
+  responses: <
+    key: "i7way2P2z-U="
+    value: <
+      struct: <
+        map_type: <
+          key: <
+            oneof: <
+              options: <
+                key: "9v8pKk6Y-tY="
+                value: <
+                  primitive: <
+                    string_value: <
+                      value: "addr1@example.com"
+                    >
+                  >
+                >
+              >
```

Comment:
Need to
1. Infer that these are e-mail addresses and
2. Remove the concrete values.

Where in our code do we do this?

Reply:
1. Is done in the client today, the library is superstar/lib/data_formats (GetFormats)
2. happens in lib/learn/sanitize.go with the sanitizingHasher

Reply:
Hard-coding map keys as unformatted strings for now (9892d00).